resources/migrations/7_v0.6.1.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "aws_iam_password_policies" ADD COLUMN IF EXISTS policy_exists boolean;
```

Comment:
this should be in v0.6.5 i.e for next release, same below

Reply:
fixed